src/jit/codegenlinear.h

```diff
@@ -42,6 +42,10 @@
     void                genPutArgStk(GenTreePtr treeNode);
     unsigned            getBaseVarForPutArgStk(GenTreePtr treeNode);
 
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
```

Comment:
Note that for fast tail call implementation we need to define this routine for each target apropriately. IMO,  this should be in codegencommon.cpp.